coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20161014-add-secondary-manifest-queue-index.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="marcs" id="20161014-add-secondary-manifest-queue-index">
+        <preConditions onFail="MARK_RAN">
```

Comment:
No, I can't seem to get indexExists to work... it just thinks the index doesn't exist and tries to create it.  There is a similar comment for an older core migration... https://github.com/socrata/core/blob/c872586a038148b2e0e602163a05df494c735b13/unobtainium/src/main/resources/config/migrations/20110311-split-index-and-accumulate.xml#L46-L46